flutter-studio/testSrc/com/android/tools/idea/tests/gui/framework/fixture/IdeaFrameFixture.java

```diff
@@ -345,7 +334,7 @@ protected void selectProjectMakeAction() {
    * @param path the series of menu names, e.g. {@link invokeActionByMenuPath("Build", "Make Project")}
    */
   public IdeaFrameFixture invokeMenuPath(@NotNull String... path) {
-    getMenuFixture().invokeMenuPath(path);
+    getMenuFixture().invokeMenuPath(10, path);
```

Comment:
is this the same as waitAndInvokeMenuPath below?

Reply:
Sorry for not answering sooner; missed the question.

It looks like it is the same. I don't write (or even read) this. Copy/paste/edit per instructions at the top. :) I would prefer to re-use IdeFrameFixture but it isn't intended to be re-used, so I have to re-write it.